### PR TITLE
Feature/smart tv

### PR DESCRIPTION
**Pull request by maniland**
adds docs for:
- Samsung Tizen TV's
- LG WebOS TV's

### CODE REVIEW

**Comment by somerobmitchell** [comment]
added @fschurer as this is a public docs change

File: docs/source/integrations/smart_tv/tizen.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Player Partners support
+
+All of our player partners provide mechanisms to make use of VUDRM, and as Tizen only supports PlayReady, the documentation for each can be followed to implement.  
+
+## Native support
+
+Tizen has an inbuilt player within the SDK which can be used to provide encrypted playback utilising PlayReady DRM.
+
+Full documentation of the `avplayer` can be found on Samsung's developer [documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/ovw008/index)
```
Comment (PerritonBranch) [inline docs/source/integrations/smart_tv/tizen.md]:
```suggestion
Full documentation of the `avplayer` can be found on Samsung's developer [documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/ovw008/index).
```

File: docs/source/integrations/smart_tv/tizen.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+implementing a PlayReady solution can be done with the following code sample.
```
Comment (PerritonBranch) [inline docs/source/integrations/smart_tv/tizen.md]:
```suggestion
Implementing a PlayReady solution can be done with the following code sample.
```

File: docs/source/integrations/smart_tv/tizen.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Full dicumentatuion of Samsungs PlayReady support can be found [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index)
```
Comment (PerritonBranch) [inline docs/source/integrations/smart_tv/tizen.md]:
```suggestion
For further information on Samsungs PlayReady support please visit the [Tizen PlayReady documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
```

File: docs/source/integrations/smart_tv/index.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
+SMART TV's
```
Comment (PerritonBranch) [inline docs/source/integrations/smart_tv/index.rst]: This needs to be added to `/docs/source/integrations/integrations.rst` so that it shows up on RTD.

Comment (maniland) [reply docs/source/integrations/smart_tv/index.rst]: done

**Comment by maniland** [comment]
ready for another look

File: docs/source/integrations/smart_tv/tizen.md (hunk 3 above)
Comment (PerritonBranch) [reply docs/source/integrations/smart_tv/tizen.md]: Can I at least have the full stop at the end of the sentence?

File: docs/source/integrations/integrations.rst [hunk 5]
```diff
@@ -8,4 +8,5 @@ INTEGRATIONS
    mobile/index.rst
    players/index.rst
    casting/index.rst
```
[the remark this answers is not in the document]
Comment (maniland) [reply docs/source/integrations/integrations.rst]: no 😛

File: docs/source/integrations/smart_tv/tizen.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Native support
+
+Tizen has an inbuilt player within the SDK which can be used to provide encrypted playback utilising PlayReady DRM.
+
+For further information on Samsungs PlayReady support please visit the [Tizen PlayReady documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
+
+Implementing a PlayReady solution can be done with the following code sample.
+
+Full dicumentatuion of Samsungs PlayReady support can be found [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
```
Comment (PerritonBranch) [inline docs/source/integrations/smart_tv/tizen.md]:
```suggestion
For Samsungs full documentation on PlayReady support please visit the [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
```

File: docs/source/integrations/smart_tv/tizen.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    function (error) {
+        console.error(error.message);
+    },
+    0
+);
+
+let customData = "<YOUR VUDRM TOKEN>"
+avplayObj.setPlayerProperty(3, customData, customData.length);
+
+let la_url ="https://playready-license.drm.technology/rightsmanager.asmx";
```
Comment (somerobmitchell) [inline docs/source/integrations/smart_tv/tizen.md]: javascript should be camalCase

**Comment by maniland** [comment]
[!giphy](https://media.giphy.com/media/7WwVYKDMt5khG/giphy.gif)

File: docs/source/integrations/smart_tv/lg.md [hunk 8]
```diff
@@ -32,9 +32,41 @@ If you choose to use the luna services please follow LG's own documentation desc
...
+    }
+    return output;
+}
+
+function getPlayReadyMessage(manifest, token) {
     let xml = new DOMParser().parseFromString(manifest, 'application/xml');
-    let playReadyEncryptedXML = xml.querySelector('pro');
-    return window.atob(playReadyEncryptedXML);
+    let pros = xml.querySelectorAll('pro');
+    pros = Array.prototype.slice.call(pros);
```
Comment (stefanovualto) [inline docs/source/integrations/smart_tv/lg.md]:
```suggestion
    pros = Array.from(pros);
```

File: docs/source/integrations/smart_tv/lg.md [hunk 9]
```diff
@@ -32,9 +32,41 @@ If you choose to use the luna services please follow LG's own documentation desc
...
+    let startingPoint = message.indexOf('<WRMHEADER');
+    message = message.substring(startingPoint, message.length);
+	return message;
+}
+
+function cleanString(input) {
+    var output = "";
+    for (var i=0; i<input.length; i++) {
+        let code = input.charCodeAt(i); //162 0-31
+        if (code !== 162 && code > 31) {
```
Comment (stefanovualto) [inline docs/source/integrations/smart_tv/lg.md]:
```suggestion
let isControlCode = code > 31;
let isWeirdCode = code === 162;
if (!isWeirdCode && isControlCode) {
```

**Pull request merged**